fuzzers/074-dump_all/create_node_tree.py

```diff
@@ -273,7 +272,7 @@ def main():
 
     print('{} Writing node tree'.format(datetime.datetime.now()))
     with open(os.path.join(args.output_dir, 'node_tree.json'), 'w') as f:
-        xjson.pprint(f, nodes)
+        json.dump(nodes, f, indent=2, sort_keys=True)
```

Comment:
Since we are going to sort + indent things at a later date, should we just skip indent / sort_keys?

Reply:
It's still convenient for debugging to have the `indent=2`, and the sort_keys should also be fast.